filebeat/tests/system/test_modules.py

```diff
@@ -190,7 +190,7 @@ def _test_expected_events(self, test_file, objects):
 
 def clean_keys(obj):
     # These keys are host dependent
-    host_keys = ["host.name", "beat.hostname", "beat.name"]
+    host_keys = ["host.name", "agent.hostname", "agent.type"]
```

Comment:
Same: "host.name" => "host.hostname"

Reply:
see above

Reply:
We still need to figure that one out. This PR keeps the same behaviour as we had.